src/components/header-logo.ts

```diff
@@ -0,0 +1,34 @@
+import { LitElement, css, html } from "lit";
+import { customElement } from "lit/decorators.js";
+
+@customElement("header-logo")
+export class HeaderLogo extends LitElement {
+  render() {
+    return html`
+      <header>
+        <img src="./images/logo.svg" alt="Wake Word Collective logo" />
+      </header>
+    `;
+  }
+
+  static styles = css`
+    :host {
+      flex: 0 !important;
+    }
```

Comment:
Why? If this is to reset some style from the parent, then the parent should reset it.